### PR TITLE
Updating upload artifact version

### PR DESCRIPTION
**Pull request by Adir111**
needed for deployment.

- - -
This change is [Reviewable](https://reviewable.io/reviews/kube-HPC/hkube/2007)

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -88,7 +88,7 @@ jobs:
           prNumber: ${{ github.event.deployment.payload.prNumber }}
           branchName: ${{ github.event.ref }}
 
-      - uses: actions/upload-artifact@v2
+      - uses: actions/upload-artifact@v3
```
Comment (RonShvarz) [inline .github/workflows/deploy.yml:91]: There are two more instances of the same action used in this file.

**Review by RonShvarz — CHANGES_REQUESTED** [review]

File: core/pipeline-driver/package.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
     "@hkube/etcd": "^5.1.9",
     "@hkube/logger": "^2.0.2",
     "@hkube/metrics": "^1.0.43",
-    "@hkube/parsers": "^2.1.9",
+    "@hkube/parsers": "^2.1.10",
```
Comment (RonShvarz) [inline core/pipeline-driver/package.json:18]: Is it part of this PR? or from another issue ?

**Review by RonShvarz — CHANGES_REQUESTED** [review]

**Review by RonShvarz — APPROVED** [review]

**Review by RonShvarz — COMMENTED** [review]
> Reviewed 1 of 1 files at r2, 2 of 2 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/kube-HPC/hkube/2007)* status: all files reviewed, 2 unresolved discussions (waiting on @Adir111)

**Review by Adir111 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/kube-HPC/hkube/2007)* status: all files reviewed, 2 unresolved discussions (waiting on @RonShvarz)

___
*[`.github/workflows/deploy.yml` line 91 at r1](https://reviewable.io/reviews/kube-HPC/hkube/2007#gh-1760015720:-O6pTrD85mWyjGmo2PcL:b-896fix) ([raw file](https://github.com/kube-HPC/hkube/blob/611f3464478738ffc0b84752af04f2d5a35a2001/.github/workflows/deploy.yml#L91)):*
<details><summary><i>Previously, RonShvarz (Ron Shvarzburd) wrote…</i></summary><blockquote>

There are two more instances of the same action used in this file. 
</blockquote></details>

Done.
___
*[`core/pipeline-driver/package.json` line 18 at r2](https://reviewable.io/reviews/kube-HPC/hkube/2007#gh-1760025291:-O6pTr_D5zjFgr2_HjI2:b-896fix) ([raw file](https://github.com/kube-HPC/hkube/blob/dbe530db41500b0722900077ed0f8cda19d9db4b/core/pipeline-driver/package.json#L18)):*
<details><summary><i>Previously, RonShvarz (Ron Shvarzburd) wrote…</i></summary><blockquote>

Is it part of this PR? or from another issue ?
</blockquote></details>

Done.

**Review by RonShvarz — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/kube-HPC/hkube/2007)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @Adir111)

**Pull request merged**